components/rich_text_editor/extensions/emoji/EmojiList.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    v-show="items.length"
+    class="dt-emoji-suggestion-list"
+  >
+    <dt-list-item
+      v-for="(item, index) in items"
+      :key="index"
+      :class="[
+        'dt-emoji-suggestion-list-item',
+        { 'is-selected': index === selectedIndex },
+      ]"
+      navigation-type="arrow-keys"
+      @click="selectItem(index)"
+      @keydown.prevent.stop="onKeyDown"
+    >
+      <dt-emoji
+        size="200"
+        :code="item"
+      />
+      <div class="dt-emoji-suggestion-list-text">
+        {{ item }}
+      </div>
+    </dt-list-item>
+  </div>
+</template>
+
+<script>
+import { DtEmoji } from '@/emoji';
+import { DtListItem } from '@/components/list_item';
+
+export default {
+  name: 'EmojiList',
+  components: {
+    DtEmoji,
+    DtListItem,
+  },
+
+  props: {
+    items: {
+      type: Array,
+      required: true,
+    },
+
+    command: {
+      type: Function,
+      required: true,
+    },
+  },
+
+  data () {
+    return {
+      selectedIndex: 0,
+    };
+  },
+
+  watch: {
+    items () {
+      this.selectedIndex = 0;
+    },
+  },
+
+  methods: {
+    onKeyDown ({ event }) {
+      if (event.key === 'ArrowUp') {
+        this.upHandler();
+        return true;
+      }
+
+      if (event.key === 'ArrowDown') {
+        this.downHandler();
+        return true;
+      }
+
+      if (event.key === 'Enter') {
+        this.enterHandler();
+        return true;
+      }
+
+      return false;
+    },
```

Comment:
the scroller isnt working properly. if you scroll with your mouse wheel it will show. Am currently trying to fix this but been spending too much time on this ticket, so planning to open a bug for now for this.